Content.Shared/PDA/PDAUpdateState.cs

```diff
@@ -34,4 +38,11 @@ public struct PDAIdInfoText
         public string? IdOwner;
         public string? JobTitle;
     }
+
+    [Serializable, NetSerializable]
+    public struct StationAlert
+    {
+        public string? Level;
+        public Color Color;
```

Comment:
shouldnt these fields be part of PDAIdInfoText

Reply:
In this case, station name must also be stored in PDAIdInfoText